### ISSUE TITLE
DMCA Copyright notice

### ISSUE THREAD

**Issue by 1-1is0** [farshadsangari/farshadsangari#1]
consider removing the same meme as my Github `README.md`. it is evident, that you copied the same image from my page.

**Comment by S-razmi** [comment]
I checked the content, and it seems that @1-1is0  is right.

**Comment by a12hsh** [comment]
@1-1is0 is right, you should remove it.

**Comment by Alireza-Ghafouri** [comment]
Seems true 😏

**Comment by 1-1is0** [comment]
Maybe you could swap it out for something else that reflects your awesome coding style? It would be a great way to show your unique personality on your profile.

Thinking about it more broadly, it kind of ties into the whole idea of code conduct within the open-source community. It's a way of keeping things respectful and maintaining a positive environment for everyone to collaborate and share.

**Comment by 1-1is0** [comment]
here is the pull request, I remove the image so you don't have to spend time doing it.
just press the big green button.
https://github.com/farshadsangari/farshadsangari/pull/2

**Comment by farshadsangari** [comment]
You are mistaken, Amir Jan! The meme you're using includes a black horizontal line at the top, which mine does not have.

**Comment by farshadsangari** [comment]
You are mistaken, Amir Jan @1-1is0 ! The meme you're using includes a black horizontal line at the top, which mine does not have.

**Comment by 1-1is0** [comment]
I'm not sure that's how the law works!
copyright protects the creator’s right to reproduce, distribute, display, perform, or adapt their creative work, as well as to create derivative works from it. [copyright.gov Copyright Basics document](https://www.copyright.gov/circs/circ01.pdf)

Unless you transform it so substantially that the original is no longer even recognizable, all you have done is prepare a “derivative work,” and only the original work’s copyright owner has the right to do that.[copyright.gov Copyright in Derivative
Works and Compilations](https://www.copyright.gov/circs/circ14.pdf).

As GitHub is a US-based company, we should follow the rule of copyright of respective country.

I hope this comment clarify everything. Hope you the best. 🙏
